ic-os/components/hostos/misc/sudoers

```diff
@@ -29,4 +29,8 @@ root	ALL=(ALL:ALL) NOPASSWD:ALL
 # Allow members of group sudo to execute any command
 %sudo	ALL=(ALL:ALL) NOPASSWD:ALL
 
+# Limited console user - restricted sudo access
+limited-console ALL=(ALL) NOPASSWD: /usr/bin/journalctl
+limited-console ALL=(ALL) NOPASSWD: /usr/bin/systemctl
```

Comment:
We could explore using polkit instead of broad access, but also, it is polkit 😅.
https://wiki.archlinux.org/title/Polkit#Allow_management_of_individual_systemd_units_by_regular_users